ui/cap-react/src/antd/App/App.js

```diff
@@ -63,8 +63,8 @@ const App = ({ initCurrentUser, loadingInit, history }) => {
               <Route path={WELCOME} component={noRequireAuth(WelcomePage)} />
               <Route path={ABOUT} component={AboutPage} />
               <Route path={POLICY} component={PolicyPage} />
+              <Route path={SCHEMA} component={SchemasPage} />
```

Comment:
should we remove from here and add it in `CMSIndex > AdminIndex` (ui/cap-react/src/components/cms/AdminIndex.js) ? so the logic/fture permission checks/etc is the same for all?